### PR TITLE
Feature/#cu-gguwtr updat--docs

### PR DESCRIPTION
**Pull request by niloufarmp**
### Changes
**Added meal selection to book request:**
Under passengers:
image

**Added selected meal to booking details response:**
image

**Following is a list of Meal SSR codes and their definition:**
https://support.travelport.com/webhelp/GWS/Content/XML_Select_Web_Service/Codes/ssr_codes.htm

### CODE REVIEW

**Comment by trip-ninja-inc** [comment]
Task linked: [CU-gguwtr update docs](https://app.clickup.com/t/gguwtr)

File: tn_api_pricing_booking_spec.yml [hunk 1]
```diff
@@ -1143,6 +1183,105 @@ components:
...
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              itinerary_reference:
+                                description: The itinerary reference to apply the card to.
+                                type: string
+                                example: "1"
+                              meal_choice:
+                                description: Choosen meal
```
Comment (mcclymont-k) [inline tn_api_pricing_booking_spec.yml]:
```suggestion
                                description: Chosen meal
```

File: tn_api_pricing_booking_spec.yml [hunk 2]
```diff
@@ -1143,6 +1183,105 @@ components:
...
+                                type: string
+                                example: '84299921292'
+                        meals:
+                          description: The passenger's meals details
+                          type: array
+                          items:
+                            type: object
+                            properties:
+                              itinerary_reference:
+                                description: The itinerary reference to apply the card to.
```
Comment (mcclymont-k) [inline tn_api_pricing_booking_spec.yml]:
```suggestion
                                description: The itinerary reference to apply the meal to.
```

File: tn_api_pricing_booking_spec.yml [hunk 3]
```diff
@@ -1362,6 +1501,96 @@ components:
...
+          type: array
+          items:
+            type: object
+            properties:
+              itinerary_reference:
+                description: The itinerary reference to apply the card to.
+                type: string
+                example: "1"
+              meal_choice:
+                description: Choosen meal
```
Comment (mcclymont-k) [inline tn_api_pricing_booking_spec.yml]:
```suggestion
                description: Chosen meal
```

File: tn_api_pricing_booking_spec.yml [hunk 4]
```diff
@@ -1362,6 +1501,96 @@ components:
...
+            type: string
+            example: '8284828284'
+        meals:
+          description: The passenger's meals details
+          type: array
+          items:
+            type: object
+            properties:
+              itinerary_reference:
+                description: The itinerary reference to apply the card to.
```
Comment (mcclymont-k) [inline tn_api_pricing_booking_spec.yml]:
```suggestion
                description: The itinerary reference to apply the meal to.
```

**Review by Iceman997 — APPROVED** [review]

**Review by mcclymont-k — APPROVED** [review]

**Pull request merged**